locales/fr.json

```diff
@@ -216,7 +216,7 @@
     "migrations_loading_migration": "Chargement de la migration {id}...",
     "migrations_migration_has_failed": "La migration {id} a échoué avec l'exception {exception} : annulation",
     "migrations_no_migrations_to_run": "Aucune migration à lancer",
-    "migrations_skip_migration": "Ignorer et passer la migration {id}...",
+    "migrations_skip_migration": "Ignorer et passer la migration {id}…",
```

Comment:
![](https://c.tenor.com/rVITf4xkgrkAAAAC/steve-carell-no.gif)

Reply:
Is there any way to display guidelines in Weblate? 😅

Reply:
yeah probably ... anyway there's this script that automatically restandardize en.json and fr.json but hmpf: https://github.com/YunoHost/yunohost/blob/dev/tests/reformat_locales.py#L36